docs/learning/guidelines/release-tags-guidelines.md

```diff
@@ -62,7 +63,13 @@ More advanced linking syntax should not be used as it would distract in the lint
 
 The `@extensions` tag indicates that an API item is officially included in the `@twin/core-extension` API and can be used to develop *iTwin.js Extensions*.
 
-> Note: only *public* API items can have an `@extensions` tag (there is an lint rule to check).
+> Note: only *public* or *preview* API items can have an `@extensions` tag (there is an lint rule to check).
+
+### @preview
```

Comment:
Did we actually implement this?

Reply:
Yes there is an eslint rule that will make sure extensions only have preview or public tags

Reply:
okay, I moreso meant do we actually have a preview Extension API. From what I remember we skipped that for now.  If so, let's not document that it could be there.

Reply:
okay, sounds good.